src/main/java/org/opentripplanner/routing/algorithm/transferoptimization/model/OptimizedPathTail.java

```diff
@@ -138,15 +141,21 @@ public OptimizedPath<T> build() {
 
   @Override
   public String toString() {
-    return ValueObjectToStringBuilder
-      .of()
-      .addObj(super.toString())
-      .addText(" [")
-      .addCostCenti(generalizedCost())
-      .addCostCenti(transferPriorityCost, "pri")
-      .addCostCenti(generalizedCostWaitTimeOptimized(), "wtc")
-      .addText("]")
-      .toString();
+    var builder = ValueObjectToStringBuilder.of().addObj(super.toString()).addText(" [");
+
+    if (generalizedCost != RaptorCostCalculator.ZERO_COST) {
+      builder.addObj(RaptorValueFormatter.formatC1(generalizedCost()));
```

Comment:
The transfer-optimizer operate on Raptor Paths (c1/centi-seconds), but I have not refacored it - I only changed the code inside raptor, to limit the scope. I used to consider it a OTP domain thing, but I am not sure any more - it is between Raptor and OTP, but since it only operate on Raptor types I tend to place it with Raptor. I think it does not matte too much, since it is its own component, so choosing c1 or generalized-cost does not matter that much as long as it is consistent. But since it is using centi-seconds for the c1/generalized-cost, using c1 as a name is probably the best chose.

Reply:
I consider this to be out-of-scope for this PR.

Reply:
Thanks, then I understand